client/wfprev-war/src/main/angular/src/app/components/list-panel/projects-list/projects-list.component.html

```diff
@@ -0,0 +1,127 @@
+<div>
+  <div class="list-header">
+    <!-- Sort Dropdown -->
+    <div class="sort-dropdown">
+      <label>Sort:</label>
+      <select id="sort" (change)="onSortChange($event)">
+        <option class="default-option" value="">select...</option>
+        <option *ngFor="let option of sortOptions" [value]="option.value">{{ option.label }}</option>
+      </select>
+    </div>
+
+    <!-- Sync Toggle -->
+    <div class="sync-toggle">
+      <div class="label-container">
+        <label>Sync with Map</label>
+      </div>
+      <div class="toggle-container">
+        <mat-slide-toggle color="primary" (change)="onSyncMapToggleChange($event)">
+          <span>Off/On</span>
+        </mat-slide-toggle>
+      </div>
+    </div>
+
+    <!-- Result Count -->
+    <div class="result-count">
+      <span>{{ resultCount }} Results</span>
+    </div>
+  </div>
+
+  <div class="list-contents">
+    <mat-accordion>
+      
+      <mat-expansion-panel *ngFor="let item of projectList" #panel="matExpansionPanel">
+        <mat-expansion-panel-header class="expansion-panel-header">
+          <mat-panel-title class="project-title">
+            <div class="custom-indicator">
+              <img
+                *ngIf="!panel.expanded"
+                src="../../../../assets/collapsed-indicator.svg"
```

Comment:
Do we really need "../../../" here? Does src="/assets/collapsed-indicator.svg" not work? It makes the code cleaner

Reply:
we dont as it is relative path... cleaned up all these and other places